script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -1143,7 +1144,8 @@ Or rather, %{i}because%{/i} the situation is so bad―――
 [sha:bdeb7b14cb7c90918a63e65a10718d7f7c7c717d]{
 -- Page 494, Offset 19509.
 -- 悪夢のような発作がやってきてしまった。
-A nightmarish seizure takes hold of her.
+// Seizure is too strong a word here and implies a loss of consciousness
+A nightmarish spasm takes hold of her.
```

Comment:
Hmm, I do think that this should be _fairly_ strong. I don't think seizing necessarily requires loss of consciousness (though of course it can).